eng/Version.Details.xml

```diff
@@ -42,56 +42,56 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>c623d96fe33cc360bd9981c9e6f6a646d7881d9e</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Build" Version="17.4.0-preview-22464-02">
```

Comment:
I'm a little worried about these version changes - does the PR need to be rebased or something to prevent incorporating bad versions?

Reply:
@baronfel Changes to `Version.Details.xml` and `Versions.props` were done by you (i.e. were there in the PR when I took it over) so I assumed they were needed. It might be worth rebasing from the branch as you suggested.

Reply:
ok cool - glad to hear it. I'll fix that up now then if that's ok?

Reply:
Sure go for it! Thank you

Reply:
@Evangelink done with the rebase. Can you take a quick look and make sure that nothing changed with your fixes?  The rebase applied cleanly, so I don't think there were any issues, but additional verification would be great.